olclient/cli.py

```diff
@@ -103,29 +103,37 @@ def main():
             raise
 
     if args.get_olid:
-        return ol.Edition.get_olid_by_isbn(args.isbn)
+        sys.stdout.write(ol.Edition.get_olid_by_isbn(args.isbn))
+        return
```

Comment:
Before this change this function returned something to the caller.  If we are going to change that then
1. Please add a type hint to the function that uses `-> Optional[]`
2. Please use `return None` to be explicit

Reply:
This seems unnecessary-- the return value from `main()` wasn't used or at least was part of the problem causing the output to be sent to STDERR.  `main()` should not return a result as far as I can see.

Reply:
Using `print()` for output in `main()` without any return values seems clearer to me. The output to stderr on errors that this PR makes seems fine though.

Reply:
@hornc 
I see what you are saying. We would be needing return in two places (in the configure section) because we want to break. In the other places we can avoid using return since its a if-elseif-else. I'll refactor that. Thanks for the suggestion!

Regarding the second point, I'm actually fine with both `sys.stdout.write` and `print`. I personally think stdout.write is more explicit. But print works too.

Reply:
Made changes @ https://github.com/internetarchive/openlibrary-client/pull/227/commits/3268b957c14b0d2808a2411f8cd35fbf2e7ca9a4 👍🏽

Reply:
@cclauss Let me know if you think there are any more changes to be done. If not, we can merge this PR 😃